### PR TITLE
Refine script execute page

### LINKED ISSUE #372: Refine ScriptExecute page

### PR DESCRIPTION
**Pull request by prin-r**
Fixes #372

### CODE REVIEW

File: scan/src/components/ScriptExecute.re [hunk 1]
```diff
@@ -55,7 +55,16 @@ module Styles = {
...
 };
 
-let parameterInput = (name, dataType, value, updateData) => {
+type selection_t = {
+  name: string,
+  dataType: string,
+  value: string,
+};
+
+let newSelection = (name, dataType, value) => {name, dataType, value};
```
Comment (sorawit) [inline scan/src/components/ScriptExecute.re]: Dont think this function is necessary.

File: scan/src/components/ScriptExecute.re [hunk 2]
```diff
@@ -116,17 +117,19 @@ let reducer = _state =>
...
+    React.useState(_ =>
+      params->Belt.List.map(({name, dataType}) => newSelection(name, dataType, ""))
+    );
   let (result, dispatch) = React.useReducer(reducer, Nothing);
 
   let updateData = (targetName, newVal) => {
     let newData =
-      data->Belt.List.map(((name, dataType, value)) =>
+      data->Belt.List.map(({name, dataType, value}) =>
         if (name == targetName) {
```
Comment (sorawit) [inline scan/src/components/ScriptExecute.re]: We can do 
```
data->Belt.List.map(selection => {
   ...selection,
   value: select.name == targetName ? newVal : select.value,
}) 
```

**Review by sorawit — CHANGES_REQUESTED** [review]

**Review by sorawit — APPROVED** [review]
LGTM thanks

File: scan/src/components/ScriptExecute.re (hunk 2 above)
Comment (prin-r) [reply scan/src/components/ScriptExecute.re]: done

File: scan/src/components/ScriptExecute.re (hunk 1 above)
Comment (prin-r) [reply scan/src/components/ScriptExecute.re]: done

**Review by evilpeach — APPROVED** [review]

**Pull request merged**